assets/src/scripts/routes/common.js

```diff
@@ -108,14 +108,52 @@ export default {
 				// Assign the button
 				let btn = heading.querySelector( 'button' );
 
-				btn.onclick = () => {
```

Comment:
this line changed to what you see in [line 136](https://github.com/pressbooks/pressbooks-book/pull/641/files#diff-5004549ff9223d9c89a0580f7c2b87c3R136).